framework/Web/UI/TTemplateManager.php

```diff
@@ -69,17 +74,25 @@ public function getTemplateByClassName($className)
 
 	/**
 	 * Loads the template from the specified file.
-	 * @param mixed $fileName
-	 * @return \Prado\Web\UI\TTemplate template parsed from the specified file, null if the file doesn't exist.
+	 * @param string $fileName The file path and name of the template.
+	 * @param string $tplClass, default null for the Default Template Class
+	 * @param string $culture culture string, null to use current application culture
+	 * @return \Prado\Web\UI\TTemplate or subclass template parsed from the specified file, null if the file doesn't exist.
 	 */
-	public function getTemplateByFileName($fileName)
+	public function getTemplateByFileName($fileName, $tplClass = null, $culture = null)
 	{
-		if (($fileName = $this->getLocalizedTemplate($fileName)) !== null) {
+		if ($tplClass === null) {
+			$tplClass = $this->_defaultTemplateClass;
+		}
+		if ($tplClass != TTemplate::class && !is_subclass_of($tplClass, TTemplate::class)) {
```

Comment:
We have an ITemplate interface that we could use. in place of this.
We'll probably need to add to the interface the method getIncludedFiles() used by TTemplate Manager for caching purposes

Reply:
Keying from ITemplate in the latest iteration.  ITemplate adds getIncludedFiles and the various ITemplates around PRADO have been updated with the new function returning [].